host/sgx/ocalls/ocalls.c

```diff
@@ -334,3 +339,26 @@ oe_result_t oe_verify_quote_ocall(
 
     return result;
 }
+
+oe_result_t oe_sgx_get_additional_host_entropy_ocall(uint8_t* data, size_t size)
+{
+    oe_result_t result = OE_FAILURE;
+
+    if (!data || !size)
+        OE_RAISE(OE_INVALID_PARAMETER);
+
+#if defined(__linux__)
+    if (getrandom((void*)data, size, 0) == -1)
```

Comment:
There are other error scenarios to handle here. Sometimes getrandom may fill the buffer partially:
https://man7.org/linux/man-pages/man2/getrandom.2.html